pages/confirm_email.tsx

```diff
@@ -5,12 +5,13 @@ import Head from 'components/head';
 import HeroBanner from 'components/HeroBanner/HeroBanner';
 import Content from 'components/Content/Content';
 import Alert from '../components/Alert/Alert';
+import type { NextPageContext } from 'next';
 
 ConfirmEmail.propTypes = {
```

Comment:
Can you remove propTypes definitions for converted files and the ESLint rules around propTypes as part of this PR?